webauthn.go

```diff
@@ -0,0 +1,339 @@
+package webauthn
+
+import "time"
+
+type (
```

Comment:
super nit: The wrapping type() block makes me think these are related / grouped types. Total preference thing but I'd probably prefer to have the type blocks separated.